index.html

```diff
@@ -160,7 +160,47 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
             <a href="https://github.com/w3c/mediacapture-region/issues/18">issue #18</a>.
           </p>
         </div>
-        <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <dl data-link-for="CropTarget" data-dfn-for="CropTarget">
+          <dt>
+            <dfn>fromElement()</dfn>
+          </dt>
+          <dd>
+            <p>
+              Calling {{CropTarget/fromElement}} with an {{Element}} of a supported type associates
+              that {{Element}} with a {{CropTarget}}. This {{CropTarget}} may be used as input to
+              {{BrowserCaptureMediaStreamTrack/cropTo}}. We define a
+              <dfn>valid CropTarget</dfn> as one returned by a previous call to
```

Comment:
This doesn't belong here which I've stated in https://github.com/w3c/mediacapture-region/issues/29#issuecomment-1074008247, but since this is in some ways a code-move, I'll accept a separate PR to remove it, in order to break down changes into smaller PRs.

Reply:
This is prose was moved unchanged. The status quo of the document has not been altered. Let's have a separate discussion for this.